index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="style.css">
     <script defer src="menu.js"></script>
     <script defer src="modal.js"></script>
+    <script defer src="validation.js"></script>
```

Comment:
- [ ] You have not made any changes and the validation does not still work. You are selecting the form using an `id` but the form has only a `class`. Kindly verify your code and make sure the validation works and the error message shows as required before submitting your project for review 👍
- Your code:
![Annotation 2022-04-15 101109](https://user-images.githubusercontent.com/85113567/163552034-b9b540fc-2e4a-47ee-9efa-755896d0b4c4.jpg)
![Annotation 2022-04-15 101206](https://user-images.githubusercontent.com/85113567/163552054-d5a8249c-970d-4afd-89fb-13cd6b0b5a73.jpg)